### PR TITLE
feat(form): Creates the Form Component.

### PR DESCRIPTION
**Pull request by Keikonichi**
Creates the Form component of the Contact page.

-The style needs to be improved to match the layout
-The data received by the form needs to be sent correctly

image

### CODE REVIEW

**Comment by Keikonichi** [comment]
I chose to leave the layout at 1024 as shown in the picture because the footer was overflowing.

image

**Comment by Keikonichi** [comment]
On the 1280 screen, the layout looks like this

image

**Comment by Keikonichi** [comment]
The task is finished.
@thiagobraga 
@renatoagomes

File: app/contato/page.tsx [hunk 1]
```diff
@@ -9,6 +12,10 @@ export default function Contact() {
         </div>
```
Comment (renatoagomes) [inline app/contato/page.tsx:12]: @Keikonichi favor corrigir o errinho de português no 'conversar'

**Review by renatoagomes — CHANGES_REQUESTED** [review]

**Pull request merged**